src/vmm/src/vmm_config/net.rs

```diff
@@ -251,6 +251,16 @@ impl NetworkInterfaceConfigs {
 
         Ok(())
     }
+
+    #[cfg(test)]
+    pub fn len(&self) -> usize {
+        self.if_list.len()
+    }
+
+    #[cfg(test)]
+    pub fn is_empty(&self) -> bool {
+        self.if_list.len() == 0
+    }
```

Comment:
nit: I think these could have been implemented in `mod tests {}` instead of here since they're only used in the tests in this module.

Reply:
It can be done and it will be cleaner, but it would require making `if_list` field of `NetworkInterfaceConfigs` public. This means that we will expose the internals of a `NetworkInterfaceConfigs`, which is currently driven through an interface, for commonly used functionalities. However, `BlockDeviceConfigs` is driven by this type of mixed interface + public internal fields, and some tests benefited from this. 

However, I prefer to drive these structs using defined interfaces, for better abstractions and as little as possible maintainable logic outside the implementation.